sematic/db/migrations/20230221004028_add_id_to_users.py

```diff
@@ -0,0 +1,44 @@
+# Standard Library
```

Comment:
Why do some of the migration in a sql migration and the rest in a py one? Why not do both in py?

Reply:
Hmm good point. I don't know, it feels like pure SQL migrations should be defined in SQL files, but that's somewhat arbitrary.